system/HTTP/URI.php

```diff
@@ -23,13 +23,15 @@ class URI
 	 * Sub-delimiters used in query strings and fragments.
 	 *
 	 * @const string
```

Comment:
Not related but is `@const` a valid phpdoc?

Reply:
I can't find the reference for `@const` in php documentor manual, remove it?

Reply:
That'd be nice. 😁

Reply:
removed https://github.com/codeigniter4/CodeIgniter4/pull/4759/commits/52a1823e589f6a9852af4efad180e8489e09403c 👍

Reply:
Thanks